yt_dlp/extractor/facebook.py

```diff
@@ -415,16 +419,20 @@ def extract_metadata(webpage):
             # in https://www.facebook.com/yaroslav.korpan/videos/1417995061575415/
             if thumbnail and not re.search(r'\.(?:jpg|png)', thumbnail):
                 thumbnail = None
-            view_count = parse_count(self._search_regex(
-                r'\bviewCount\s*:\s*["\']([\d,.]+)', webpage, 'view count',
-                default=None))
+            view_count = parse_count(
+                self._search_regex(
+                    (r'\bviewCount\s*:\s*["\']([\d,.]+)', r'video_view_count":\s*(\d+)',), webpage, 'view count', default=None))
+            live_view_count = (
+                get_first(post, ('video', 'liveViewerCount'), expected_type=int)
+                or get_first(post, (..., ..., 'attachments', ..., 'media', 'liveViewerCount'), expected_type=int))
             info_dict = {
                 'description': description,
                 'uploader': uploader,
                 'uploader_id': uploader_data.get('id'),
                 'timestamp': timestamp,
                 'thumbnail': thumbnail,
                 'view_count': view_count,
+                'live_view_count': live_view_count,
```

Comment:
I think my suggestions condense `view_count` and `live_view_count` enough that they could be inlined

Reply:
Changed